catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/builder/AttributeValueNormalizer.java

```diff
@@ -45,7 +45,7 @@
   // Primary purpose is to protect persistence layer from bad symbols leaking into valid prop names
   // https://codice.atlassian.net/wiki/spaces/CODICE/pages/77627393/Taxonomy+Guidelines
   private static final Pattern EXPECTED_ATTRIBUTE_NAME_PATTERN =
-      Pattern.compile("(ext\\.)?(\\p{Alpha}+\\.)?\\p{Alpha}+(-\\p{Alpha}+)*");
+      Pattern.compile("(\\p{Alpha}+([-.]\\p{Alpha}+)*)+");
```

Comment:
Regex can be read as: 
- One or more letters, upper or lower case.
- Followed by a delimiter (either a dot `.` or dash `-`) and one or more letters, which all together can occur zero or more times.
- (hmm...) and the entire thing can occur one or more times...wait, what

✏️  Now that I'm reviewing this I think the outer wrapped "one or more" grouping isn't actually necessary.